cdap-data-fabric/src/main/java/co/cask/cdap/spi/data/StructuredTableContext.java

```diff
@@ -19,15 +19,15 @@
 import co.cask.cdap.spi.data.table.StructuredTableId;
 
 /**
- * This interface provides methods that instantiate a {@link StructuredTable} during the runtime
+ * This interface provides methods that instantiate a {@link StructuredTable} during the runtime.
  * of a program.
  */
 public interface StructuredTableContext {
 
   /**
    * Get an instance of the specified Dataset.
    *
-   * @param tableId the table id
+   * @param tableId the table id.
```

Comment:
In @param you should not use periods at the end. Only in the main paragraph. See, for example, https://dev.liferay.com/participate/javadoc-guidelines TOP TIPS section.